docs/tutorials/01_introductory/viz_earth_animation.py

```diff
@@ -0,0 +1,165 @@
+"""
+===============
+Texture Sphere Animation
+===============
+In this tutorial, we will show how to animated a textured sphere.
+"""
+
+import numpy as np
+from fury import window, actor, utils, primitive, io
+import itertools
+from fury.data.fetcher import read_viz_textures, fetch_viz_textures
+
+##############################################################################
+# Create a scene to start.
+
+scene = window.Scene()
+
+##############################################################################
+# Next, load in a texture for each of the actors. For this tutorial, we will
+# be creating one textured sphere for the Earth, and another for the moon.
+# Collect the Earth texture from the FURY github using ``fetch_viz_textures``
+# and  ``read_viz_textures``, then use ``io.load_image`` to load in the
+# image.
+
+# fetch_viz_textures()
+# filename = read_viz_textures("1_earth_8k.jpg")
+# image = io.load_image(filename)
+
+fetch_viz_textures()
```

Comment:
Download failure.
HTTP Error : Not Found

Reply:
Same issue

Reply:
Thank you both for this feedback! I am working on fixing the sha256 codes for the new textures, which will hopefully solve this issue and allow them to successfully download with ``fetch_viz_textures``.